File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void)
{
        int n;
    //Getting size for pyramid
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);
    //creating height and width for half pyramids using spaces
    for (int i = 0; i < n; i = i+1)
    {
        for (int j = 0; j < n; j = j+1)
        {
            if (i + j < n - 1)
                printf(" ");
            else
                printf("#");  
        }
            printf("  ");
            for (int j = 0; j < n; j = j+1)
                {
                    if (n - i < j + 2)
                        printf("#");
                }       
                printf("\n");
    }
}

File: pset6/mario/more/mario.py
from cs50 import get_int

n = get_int("Height: ")

while (n < 1 or n > 8):
    n = get_int("Height: ")
for i in range(n):
    print(" " * (n-1-i), end="")
    print("#" * (i+1), end="  ")
    print("#" * (i+1), end="")
    print(" " * (n-1-i))